### PR TITLE
Cldc 647 lettype derived variable

### PR DESCRIPTION
**Pull request by kosiakkatrina**
Derive `lettype` value from the `renttype`, `needstype` and `providertype` fields

### CODE REVIEW

File: app/admin/organisations.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@
     selectable_column
     id_column
     column :name
-    column :org_type
+    column :providertype
```
Comment (baarkerlounger) [inline app/admin/organisations.rb]: Should probably give this a readable name?

File: app/models/organisation.rb [hunk 2]
```diff
@@ -3,6 +3,9 @@ class Organisation < ApplicationRecord
   has_many :owned_case_logs, class_name: "CaseLog", foreign_key: "owning_organisation_id"
   has_many :managed_case_logs, class_name: "CaseLog", foreign_key: "managing_organisation_id"
 
+  include DbEnums
+  enum providertype: DbEnums.providertype, _suffix: true
```
Comment (baarkerlounger) [inline app/models/organisation.rb]: Think the `display_attributes` method here will also need updating since it refers to `org_type` still?

**Comment by baarkerlounger** [comment]
Does anything need changing on the bulk upload?

File: app/admin/organisations.rb (hunk 1 above)
Comment (kosiakkatrina) [reply app/admin/organisations.rb]: Yeah, was just wondering if we need it to be the same as in the data dictionary for core?

Comment (baarkerlounger) [reply app/admin/organisations.rb]: We don't but I think consistency with the data dictionary is fine anyway. Can give this column a different name from the field name by passing a string though. Look at the one below.

File: app/models/organisation.rb (hunk 2 above)
Comment (kosiakkatrina) [reply app/models/organisation.rb]: Yes! Missed it, thanks!

File: app/admin/organisations.rb (hunk 1 above)
Comment (kosiakkatrina) [reply app/admin/organisations.rb]: Ah, that's interesting! Let me check it out

**Comment by kosiakkatrina** [comment]
> Does anything need changing on the bulk upload?

@baarkerlounger Don't think so 🤔

File: app/admin/organisations.rb [hunk 3]
```diff
@@ -17,7 +17,7 @@
     selectable_column
     id_column
     column :name
-    column :org_type
+    column "org_type", :providertype
```
Comment (baarkerlounger) [inline app/admin/organisations.rb]: Would suggest "Org type" or "Organisation type" so it's humanized, consistent with other table labels.

**Review by baarkerlounger — APPROVED** [review]

**Pull request merged**